diktat-rules/src/test/resources/test/smoke/src/main/kotlin/script/SimpleRunInScriptExpected.kts

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
discussed, need to add test for kts with package name

Reply:
Added